2025-02-24T21:41:13.3970906Z ##[group]Run pytest tardis --tardis-regression-data=/home/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append -m "not continuum"
2025-02-24T21:41:13.3972254Z [36;1mpytest tardis --tardis-regression-data=/home/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append -m "not continuum"[0m
2025-02-24T21:41:13.4027973Z shell: /usr/bin/bash -l {0}
2025-02-24T21:41:13.4028439Z env:
2025-02-24T21:41:13.4028617Z   CACHE_NUMBER: 0
2025-02-24T21:41:13.4029231Z   PYTEST_FLAGS: --tardis-regression-data=/home/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append
2025-02-24T21:41:13.4029858Z   CODECOV_TOKEN: 
2025-02-24T21:41:13.4030079Z   MAMBA_ROOT_PREFIX: /home/runner/micromamba
2025-02-24T21:41:13.4030376Z   MAMBA_EXE: /home/runner/micromamba-bin/micromamba
2025-02-24T21:41:13.4030712Z   CONDARC: /home/runner/work/_temp/setup-micromamba/.condarc
2025-02-24T21:41:13.4031059Z ##[endgroup]
2025-02-24T21:41:23.0808773Z /home/runner/work/tardis/tardis/tardis/io/model/readers/tests/test_artis_readers.py:61: SyntaxWarning:
2025-02-24T21:41:23.0809505Z 
2025-02-24T21:41:23.0809722Z assertion is always true, perhaps remove parentheses?
2025-02-24T21:41:23.0810074Z 
2025-02-24T21:41:28.4184078Z /home/runner/work/tardis/tardis/tardis/io/model/readers/tests/test_artis_readers.py:61: SyntaxWarning:
2025-02-24T21:41:28.4184879Z 
2025-02-24T21:41:28.4185111Z assertion is always true, perhaps remove parentheses?
2025-02-24T21:41:28.4185472Z 
2025-02-24T21:41:29.9330825Z ============================= test session starts ==============================
2025-02-24T21:41:29.9331569Z platform linux -- Python 3.12.4, pytest-8.2.2, pluggy-1.5.0
2025-02-24T21:41:29.9332165Z rootdir: /home/runner/work/tardis/tardis
2025-02-24T21:41:29.9332611Z configfile: pyproject.toml
2025-02-24T21:41:29.9333151Z plugins: metadata-3.1.1, anyio-4.4.0, doctestplus-1.2.1, cov-5.0.0, html-4.1.1
2025-02-24T21:41:29.9333888Z collected 1796 items / 1 error / 2 deselected / 1794 selected
2025-02-24T21:41:29.9334281Z 
2025-02-24T21:41:29.9334466Z ==================================== ERRORS ====================================
2025-02-24T21:41:29.9335126Z _____ ERROR collecting tardis/io/model/readers/tests/test_artis_readers.py _____
2025-02-24T21:41:29.9336102Z ImportError while importing test module '/home/runner/work/tardis/tardis/tardis/io/model/readers/tests/test_artis_readers.py'.
2025-02-24T21:41:29.9337183Z Hint: make sure your test modules/packages have valid Python names.
2025-02-24T21:41:29.9337997Z Traceback:
2025-02-24T21:41:29.9338568Z ../../../micromamba/envs/tardis/lib/python3.12/importlib/__init__.py:90: in import_module
2025-02-24T21:41:29.9339298Z     return _bootstrap._gcd_import(name[level:], package, level)
2025-02-24T21:41:29.9339939Z tardis/io/model/readers/tests/test_artis_readers.py:6: in <module>
2025-02-24T21:41:29.9340516Z     from tardis.io.model.readers.artis import (
2025-02-24T21:41:29.9341622Z E   ImportError: cannot import name 'read_artis_mass_fractions' from 'tardis.io.model.readers.artis' (/home/runner/work/tardis/tardis/tardis/io/model/readers/artis.py)
2025-02-24T21:41:29.9342734Z ------------------------------- Captured stdout --------------------------------
2025-02-24T21:41:29.9343559Z [[1mpy.warnings         [0m][[1;33mWARNING[0m]  
2025-02-24T21:41:29.9344458Z 	/home/runner/work/tardis/tardis/tardis/io/model/readers/tests/test_artis_readers.py:61: PytestAssertRewriteWarning:
2025-02-24T21:41:29.9345188Z 
2025-02-24T21:41:29.9345388Z assertion is always true, perhaps remove parentheses?
2025-02-24T21:41:29.9345712Z 
2025-02-24T21:41:29.9345884Z  ([1mwarnings.py[0m:112)
2025-02-24T21:41:29.9346244Z =============================== warnings summary ===============================
2025-02-24T21:41:29.9346740Z tardis/io/configuration/config_validator.py:6
2025-02-24T21:41:29.9347190Z tardis/io/configuration/config_validator.py:6
2025-02-24T21:41:29.9348116Z   /home/runner/work/tardis/tardis/tardis/io/configuration/config_validator.py:6: DeprecationWarning:
2025-02-24T21:41:29.9349236Z   
2025-02-24T21:41:29.9351108Z   jsonschema.RefResolver is deprecated as of v4.18.0, in favor of the https://github.com/python-jsonschema/referencing library, which provides more compliant referencing behavior as well as more flexible APIs for customization. A future release will remove RefResolver. Please file a feature request (on referencing) if you are missing an API for the kind of customization you need.
2025-02-24T21:41:29.9353193Z 
2025-02-24T21:41:29.9353567Z ../../../micromamba/envs/tardis/lib/python3.12/site-packages/jupyter_client/connect.py:22
2025-02-24T21:41:29.9354621Z   /home/runner/micromamba/envs/tardis/lib/python3.12/site-packages/jupyter_client/connect.py:22: DeprecationWarning:
2025-02-24T21:41:29.9355406Z   
2025-02-24T21:41:29.9355779Z   Jupyter is migrating its paths to use standard platformdirs
2025-02-24T21:41:29.9356362Z   given by the platformdirs library.  To remove this warning and
2025-02-24T21:41:29.9357005Z   see the appropriate new directories, set the environment variable
2025-02-24T21:41:29.9357775Z   `JUPYTER_PLATFORM_DIRS=1` and then run `jupyter --paths`.
2025-02-24T21:41:29.9358372Z   The use of platformdirs will be the default in `jupyter_core` v6
2025-02-24T21:41:29.9358766Z 
2025-02-24T21:41:29.9358920Z tardis/plasma/properties/__init__.py:17
2025-02-24T21:41:29.9359607Z   /home/runner/work/tardis/tardis/tardis/plasma/properties/__init__.py:17: DeprecationWarning:
2025-02-24T21:41:29.9360284Z   
2025-02-24T21:41:29.9361241Z   The numpy.linalg.linalg has been made private and renamed to numpy.linalg._linalg. All public functions exported by it are available from numpy.linalg. Please use numpy.linalg.LinAlgError instead.
2025-02-24T21:41:29.9362249Z 
2025-02-24T21:41:29.9362533Z -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
2025-02-24T21:41:29.9362939Z 
2025-02-24T21:41:29.9363175Z ---------- coverage: platform linux, python 3.12.4-final-0 -----------
2025-02-24T21:41:29.9363738Z Coverage HTML written to dir coverage_html_report
2025-02-24T21:41:29.9364188Z Coverage XML written to file coverage.xml
2025-02-24T21:41:29.9364473Z 
2025-02-24T21:41:29.9364648Z =========================== short test summary info ============================
2025-02-24T21:41:29.9365183Z ERROR tardis/io/model/readers/tests/test_artis_readers.py
2025-02-24T21:41:29.9365765Z !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
2025-02-24T21:41:29.9366328Z ================== 2 deselected, 4 warnings, 1 error in 9.19s ==================
2025-02-24T21:41:30.6529188Z ##[error]Process completed with exit code 2.
